docs/astro/src/content/docs/guide/language/concepts/slint-language.mdx

```diff
@@ -4,4 +4,150 @@ title: Slint Language
 description: Slint Language
 ---
 
-Component basics. It's components all the way down...
\ No newline at end of file
+import Link from '/src/components/Link.astro';
+
+The concepts section of the guide are to give you an overview of the language from a high level. If you want to
+dive straight in then the details then the <Link type="slintFile" label="coding section"/> and language reference
+sections are the best places to start.
+
+This section is to give you an insight into the thinking behind the language and the core concepts it is made up from.
+
+
+
+## Why Slint?
+
+The Slint language helps you to describe the User Interface of your application in a declarative way.
```

Comment:
Suggestion for more direct and actionable phrasing. We're not just here to "help", this _is_ the way we want you to do it :)

```suggestion
With the Slint language you describe the User Interface of your application in a declarative way.
```

or

```suggestion
The Slint language describes your application's User Interface in a declarative way.
```